sdk/core/azure-core/CHANGELOG.md

```diff
@@ -1,38 +1,37 @@
 # Release History
 
-## 1.0.0-beta.3 (Unreleased)
+## 1.0.0-beta.3 (20/11/10)
 
 ### Breaking Changes
 
 - `Azure::Core::Http::Url::AppendPath` now does not encode the input by default.
 - Removed `azure.hpp`.
 
+### Bug Fixes
+
+- Prevent pipeline of length zero to be created.
+- Avoid re-using a connection when a request to upload data fails while using the `CurlTransport`.
```

Comment:
Bug fix?
https://github.com/Azure/azure-sdk-for-cpp/pull/730

Reply:
Another important bug fix to call out:
https://github.com/Azure/azure-sdk-for-cpp/pull/868

Reply:
And this?
https://github.com/Azure/azure-sdk-for-cpp/pull/877

Reply:
#730 -> part of previous release
#703 -> part of last release (don't go before previous release date, this is from Oct 1st)

#868 -> it is part of the `other changes and improvements`
#877 -> Not a breaking change. We we throwing before. We just add a message for the exception

Reply:
> #730 -> part of previous release

The last azure core release was on Oct 9:
https://github.com/Azure/azure-sdk-for-cpp/commit/82079de190006690a8d1cb950ea5c1bb3b62e9e7
https://github.com/Azure/azure-sdk-for-cpp/releases/tag/azure-core_1.0.0-beta.2

Wasn't 730 merged on October 12?

> #703 -> part of last release (don't go before previous release date, this is from Oct 1st)

It got merged on October 9, **after** the beta.2 release

Please look at the commit shas after the beta.2 release. If my shas are wrong, please let me know:
https://github.com/Azure/azure-sdk-for-cpp/compare/82079de190006690a8d1cb950ea5c1bb3b62e9e7...f99e4a57c7528ef6138686f9b3e0c9e8d87cf635

Reply:
> #877 -> Not a breaking change. We we throwing before. We just add a message for the exception

Good point. Ignore that.

Reply:
> #868 -> it is part of the other changes and improvements

That's a bug fix. It should move to that category.